Add rubocop

holywyvern opened the pull request:
# Summary

- Added Rubocop gem into the Gemfile
- Added magical comments suggested by Rubocop
- Added custom Rubocop configuration for this project

# Trello Card

https://trello.com/c/D8K0qyQh/8-configurar-rubocop

epintos commented:
@rominariedel This PR should be created against the development branch so I can see the difference with that one. You could re open this PR with those changes or wait until the other PR is merged

Commit:
Added rubocop to project

# Summary

- Added rubocop to Gemfile
- Created .rubocop.yml configuration

# Trello card

https://trello.com/c/D8K0qyQh/8-configurar-rubocop

Commit:
Modified files according rubocop warnings

# Summary

- Added frozen string magical comment to files

juanisierra commented on training-app/.rubocop.yml:
```diff
@@ -0,0 +1,5 @@
+Documentation:
+  Enabled: false
+LineLength:
+  Max: 99 
```
Add
```
Style/FrozenStringLiteralComment:
  Enabled: false
```
to avoid the comments in every file

juanisierra commented on training-app/app/channels/application_cable/connection.rb:
```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```
Remove the comment in all files please

Commit:
Removed frozen_literal_string

Commit:
removed frozen literal string in files
